ts/src/pro/krakenfutures.ts

```diff
@@ -0,0 +1,1281 @@
+         * @see https://docs.futures.kraken.com/#websocket-api-public-feeds-book
+         * @param {string} symbol unified symbol of the market to fetch the order book for
+         * @param {int|undefined} limit not used by krakenfutures watchOrderBook
+         * @param {object} params extra parameters specific to the krakenfutures api endpoint
+         * @returns {object} A dictionary of [order book structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-book-structure} indexed by market symbols
+         */
+        const orderbook = await this.subscribePublic ('book', [ symbol ], params);
+        return orderbook.limit ();
+    }
+
+    async watchOrders (symbol: string = undefined, since: Int = undefined, limit: Int = undefined, params = {}) {
+        /**
+         * @method
+         * @name krakenfutures#watchOrders
+         * @description watches information on multiple orders made by the user
+         * @see https://docs.futures.kraken.com/#websocket-api-private-feeds-open-orders
+         * @see https://docs.futures.kraken.com/#websocket-api-private-feeds-open-orders-verbose
+         * @param {string|undefined} symbol not used by krakenfutures watchOrders
+         * @param {int|undefined} since not used by krakenfutures watchOrders
+         * @param {int|undefined} limit not used by krakenfutures watchOrders
+         * @param {object} params extra parameters specific to the krakenfutures api endpoint
+         * @returns {[object]} a list of [order structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
+         */
+        await this.loadMarkets ();
+        const name = 'open_orders';
+        const orders = await this.subscribePrivate (name, params);
+        if (this.newUpdates) {
+            limit = orders.getLimit (symbol, limit); // TODO: shouldn't be restricted to 1 symbol
+        }
+        return this.filterBySinceLimit (orders, since, limit, 'timestamp', true);
+    }
+
+    async watchMyTrades (symbol: string = undefined, since: Int = undefined, limit: Int = undefined, params = {}) {
+        /**
+         * @method
+         * @name krakenfutures#watchMyTrades
+         * @description watches information on multiple trades made by the user
+         * @see https://docs.futures.kraken.com/#websocket-api-private-feeds-fills
+         * @param {string} symbol unified market symbol of the market orders were made in
+         * @param {int|undefined} since the earliest time in ms to fetch orders for
+         * @param {int|undefined} limit the maximum number of  orde structures to retrieve
+         * @param {object} params extra parameters specific to the kucoin api endpoint
+         * @returns {[object]} a list of [trade structures]{@link https://docs.ccxt.com/#/?id=trade-structure}
+         */
+        await this.loadMarkets ();
+        const name = 'fills';
+        const orders = await this.subscribePrivate (name, params);
```

Comment:
you meant trades here ?

Reply:
I did not, the [subscription feed](https://docs.futures.kraken.com/#websocket-api-private-feeds-fills) is named fills

Reply:
@samgermain I was just referring to the variable name 😅